web/src/auth/ForgetPage.js

```diff
@@ -166,7 +167,7 @@ class ForgetPage extends React.Component {
     values.userOwner = this.state.application?.organizationObj.name;
     UserBackend.setPassword(values.userOwner, values.username, "", values?.newPassword).then(res => {
       if (res.status === "ok") {
-        Setting.redirectToLoginPage(this.state.application);
+        Setting.redirectToLoginPage(this.state.application, this.props.history);
```

Comment:
If set password successfully, when I use the account to login. Show errer message
sign out before login. Maybe can redirect to the home page.